src/ucc/api/ucc.h

```diff
@@ -507,7 +507,7 @@ typedef enum {
  *
  */
 typedef enum {
-    UCC_NO_SYNC_COLLECTIVES = 0, /*!< Synchornous collectives */
+    UCC_NO_SYNC_COLLECTIVES = 0, /*!< Synchronous collectives */
     UCC_SYNC_COLLECTIVES    = 1  /*!< Non-synchronous collectives */
```

Comment:
are these comments actually on the wrong lines, too?

Reply:
Yes.